src/Lachain.Core/RPC/HTTP/Web3/BlockchainServiceWeb3.cs

```diff
@@ -452,7 +452,7 @@ public JObject GetValidatorInfo(string publicKeyStr)
 
             var balance = _stateManager.CurrentSnapshot.Balances.GetBalance(addressUint160);
 
-            var stake = _systemContractReader.GetStake(addressUint160).ToMoney().ToWei();
+            var stake = _systemContractReader.GetStake(addressUint160).ToMoney();
```

Comment:
Why is it better to return it in LA not wei?

Reply:
The balance is also returned in LA, so it is easy to compare, I guess. And in fe_account api, it is returned in LA. So I thought they would be similar.

Reply:
ok